### ISSUE TITLE
[2.01] `if_then` and `evaluates_to_true` tests don’t run

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Rulesets#128]
As [stated in this comment](https://github.com/IATI/IATI-Rulesets/pull/93#discussion_r253265770), there’s a misplaced curly brace that means all the declared `if_then` and `evaluates_to_true` tests won’t run.

The [ruleset spec](https://github.com/IATI/IATI-Rulesets/blob/version-2.01/SPEC.rst#iati-ruleset-spec) shows the ruleset structure as:

```
{
    "CONTEXT": {
        "RULE_NAME": {
            "cases": CASE_DICT_ARRAY
        }
    }
}
```

…but in the case of `if_then` and `evaluates_to_true`, the `context` is missing.

So for example, try:

```sh
$ python testrules.py rulesets/standard.json meta_tests/evaluates_to_true_bad.xml
```

^^ We expect evaluates_to_true_bad.xml to fail an `evaluates_to_true` test. While it does fail various tests, crucially it doesn’t fail `evaluates_to_true`, because the test isn’t being run.

**Comment by andylolz** [comment]
Fixed in #129.

**Comment by samuele-mattiuzzo** [comment]
Thanks Andy
